### PR TITLE
Ignore docs changes for yb-ctl  GH action

### PR DESCRIPTION
**Pull request by iSignal**
yb-ctl workflow was being run for updates to the docs pages for it.

### CODE REVIEW

**Review by polarweasel — CHANGES_REQUESTED** [review]
In order to test that the ignore is working, I suggest making a change to `docs/content/latest/admin/yb-ctl.md` to make sure it doesn't trigger the CTL checks.

Deleting line 18 (one of two blank lines in a row) would be a great change, since it should be deleted anyway. :)

File: .github/workflows/yb-ctl-test.yml [hunk 1]
```diff
@@ -8,13 +8,17 @@ on:
       - master
     paths:
       - '**/yb-ctl*'
+    paths-ignore:
+       - 'docs/**'
```
Comment (polarweasel) [inline .github/workflows/yb-ctl-test.yml:12]: Need to adjust the syntax. Can't have `paths` and `paths-ignore` together.
```suggestion
      - '**/yb-ctl*'
      - '!docs/**'
```

File: .github/workflows/yb-ctl-test.yml [hunk 2]
```diff
@@ -8,13 +8,17 @@ on:
...
 
   pull_request:
     branches:
       - master
     paths:
       - '**/yb-ctl*'
-
+    paths-ignore:
+       - 'docs/**'
```
Comment (polarweasel) [inline .github/workflows/yb-ctl-test.yml:20]: Same as previous...
```suggestion
      - '**/yb-ctl*'
      - '!docs/**'
```

File: .github/workflows/yb-ctl-test.yml [hunk 3]
```diff
@@ -8,13 +8,17 @@ on:
...
     paths:
       - '**/yb-ctl*'
+    paths-ignore:
+       - 'docs/**'
 
   pull_request:
     branches:
       - master
     paths:
       - '**/yb-ctl*'
```
Comment (polarweasel) [inline .github/workflows/yb-ctl-test.yml:18]: Same as previous...
```suggestion
      - '**/yb-ctl*'
      - '!docs/**'
```

File: .github/workflows/yb-ctl-test.yml (hunk 2 above)
Comment (polarweasel) [inline .github/workflows/yb-ctl-test.yml:20]:
```suggestion
```

**Review by polarweasel — APPROVED** [review]
This looks good to me, but I made the last change here, so @iSignal please take a look and merge if it's good with you?

**Comment by iSignal** [comment]
looks good, thanks for fixing it.

**Pull request merged**